src/vs/workbench/contrib/terminal/browser/xterm/quickFixAddon.ts

```diff
@@ -228,9 +223,49 @@ export class TerminalQuickFixAddon extends Disposable implements ITerminalAddon,
 			updateLayout(this._configurationService, e);
 			this._audioCueService.playAudioCue(AudioCue.terminalQuickFix);
 			this._register(dom.addDisposableListener(e, dom.EventType.CLICK, () => {
-				this._contextMenuService.showContextMenu({ getAnchor: () => e, getActions: () => fixes, autoSelectFirstItem: true });
+				const rect = e.getBoundingClientRect();
+				const anchor = {
+					x: rect.x,
+					y: rect.y,
+					width: rect.width,
+					height: rect.height
+				};
+				// TODO: What's documentation do? Need a vscode command?
```

Comment:
documentation is shown when `fromLightbulb` is false. We will eventually want to use this to show the source etc.

Reply:
it doesn't do anything at the moment - should I remove?

Reply:
For code actions, documentation shows in the action bar below the fixes. Not sure the terminal is using it though